gadgets/trace_oomkill/README.mdx

```diff
@@ -65,7 +72,29 @@ Running the gadget:
         command terminated with exit code 137
         ```
 
-        Go back to *the first terminal* and see:
+        First check if pod was restarted:
+
+        ```bash
+        $ kubectl get pod -l app=oomkill-demo -n oomkill-demo
+        NAME                           READY   STATUS    RESTARTS   AGE
+        oomkill-demo-f5976f447-pr486   1/1     Running   0          70s
+
+        Now, check if the OOM kill event was reported by Kubernetes:
+
+        ```bash
+        $ kubectl get events --field-selector=reason=OOMKilled -n oomkill-demo
+        No resources found in oomkill-demo namespace.
+        ```
+
+        or check the `lastState` of the pod:
+
+        ```bash
+        $ kubectl get pod -l app=oomkill-demo -n oomkill-demo -o jsonpath="{.items[*].status.containerStatuses[*].lastState}"
+        {}
+        ```
+
+        Since the OOM kill event is not reported by Kubernetes, we can use the gadget to trace the OOM kill event at process level. So, go back to *the first terminal* and see:
```

Comment:
I suggest no to use the terminology "pod level" and "process level", but something simpler:

```suggestion
        Even if the OOM kill event is not reported by Kubernetes, it's still notified by the `trace_oomkill` gadget. So, go back to *the first terminal* and see:
```